packages/server/src/manifest-builder.ts

```diff
@@ -28,10 +28,16 @@ function* updateSourceMapURL(filePath: string, sourcemapName: string){
   let onRead = yield on(readStream, 'data');
   let currentURL = yield onRead.next();
```

Comment:
You should use `once` here, which accomplishes the same thing without needing to go through an event stream. Instead, it just gets the aguments of the `data` event:

```ts
let [currentURL] = yield once(readStream, 'data');
```